src/psyclone/psyir/backend/c.py

```diff
@@ -408,4 +413,40 @@ def codeblock_node(self, node):
         :raises VisitorError: The CodeBlock can not be translated to C.
 
         '''
+        # pylint: disable=no-self-use,unused-argument
```

Comment:
So `node` should be replaced with `_`...

Reply:
I can't really agree with that one. It's the base class, and even though it does not use it, it's abstract and and classes inheriting should be able to use this as a template - it's just better documentation. So I really prefer to disable the pylint warnings (or do you want to use _ instead of self as well :grinning: ? We obviously need self in order to overwrite that function).

Reply:
Indeed if I add an abstract method to base.py, we don't get the pylint warnings, see e2282c6bdca27a4d4c97f7d319670c3d073328c8 where I removed a few #pragmas by adding abstract method (oh, and I moved one identical function from c.py and fortran.py to base). The only question is: do we then want to:

1. add abstract methods for every method that is implemented in a backend language class? 
2. Or only those that trigger warnings? 
3. Or for none, and we just accept a few directives to disable pylint warnings?

That might need a vote. First one is probably the cleanest one, but quite a bit of additional typing work ;) The last one is what we have so far. And 2. is ... an odd mix :)

Reply:
We've just discussed this in the office and we disagree with your initial comment ;-) This is *not* the base class (it's the C backend) and we've provided this concrete method in order to produce a more tailored error message. We therefore all agree that using `_` for an un-used argument is the right thing to do in this case.

Reply:
OK, replaced with _, and then also remove the documentation of the parameter. I assume that was the intention?